osf/models/analytics.py

```diff
@@ -55,17 +55,21 @@ def increment(cls, user_id, action, date_string):
         return True
 
 
-class PageCounter(BaseModel):
-    primary_identifier_name = '_id'
+class PageCounter(BaseModel, ObjectIDMixin):
+    _id = models.CharField(max_length=50, default=generate_object_id, unique=True, db_index=True)
```

Comment:
To not waste time writing the _id field as before...`action:node_id:file_id:version`, just inherited from ObjectIDmixin.  Left existing ids as is (which is why I override _id so I could bump up max length.  Do we expose this __id anywhere?  Is it even needed now?